"Cannot find module 'node-zopfli'" error

After the latest update I get "Cannot find module 'node-zopfli'" error.
I think it is related with this issue. #123

Platform : Windows 10
Node version : 8.4.0
Npm version : 5.3.0
Python version : 2.7.13

```
[Browsersync] Proxying: http://localhost:8001
[Browsersync] Access URLs:
 ------------------------------------
       Local: http://localhost:8000
    External: http://192.168.1.5:8000
 ------------------------------------
          UI: http://localhost:3001
 UI External: http://192.168.1.5:3001
 ------------------------------------
module.js:491
    throw err;
    ^

Error: Cannot find module 'node-zopfli'
    at Function.Module._resolveFilename (module.js:489:15)
    at Function.Module._load (module.js:439:25)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Development\Workspaces\TESTS\Universal\patrickmichalina\fusebox-angular-universal-starter\node_modules\shrink-ray\index.js:31:14)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)

Typechecker plugin(async) :App typechecker.
Time:Sun Sep 03 2017 23:58:43 GMT+0300 (Russia TZ 2 Standard Time)
All good, no errors :-)
Typechecking time: 15738ms
Quiting typechecker

killing worker
```

--- a/src/server/server.ts
+++ b/src/server/server.ts
@@ -8,7 +8,6 @@ import * as morgan from 'morgan'
 import * as favicon from 'serve-favicon'
 import * as cookieParser from 'cookie-parser'
 import * as getOs from 'getos'
-import * as compression from 'compression'
 import ms = require('ms')
 import { createLogger } from '@expo/bunyan'
 import { join, resolve } from 'path'
@@ -20,7 +19,6 @@ import { exists, existsSync } from 'fs'
 import { EnvConfig } from '../../tools/config/app.config'
 declare var __process_env__: EnvConfig
 
-const shrinkRay = require('shrink-ray')
 const minifyHTML = require('express-minify-html')
 const bunyanMiddleware = require('bunyan-middleware')
 
@@ -57,9 +55,9 @@ app.use(cookieParser())
 getOs((err, os) => {
   if (err) throw err
   if (os.os === 'win32') {
-    app.use(compression())
+    app.use(require('compression')())
   } else {
-    app.use(shrinkRay())
+    app.use(require('shrink-ray')())
   }
 })
 

